xarray/core/resample_cftime.py

```diff
@@ -84,22 +84,17 @@ def __init__(
         self.freq = to_offset(freq)
         self.origin = origin
 
-        if isinstance(self.freq, MonthEnd | QuarterEnd | YearEnd):
-            if closed is None:
-                self.closed = "right"
-            else:
-                self.closed = closed
-            if label is None:
-                self.label = "right"
-            else:
-                self.label = label
-        # The backward resample sets ``closed`` to ``'right'`` by default
-        # since the last value should be considered as the edge point for
-        # the last bin. When origin in "end" or "end_day", the value for a
-        # specific ``cftime.datetime`` index stands for the resample result
-        # from the current ``cftime.datetime`` minus ``freq`` to the current
-        # ``cftime.datetime`` with a right close.
-        elif self.origin in ["end", "end_day"]:
+        if (
+            isinstance(self.freq, MonthEnd | QuarterEnd | YearEnd)
+            or
+            # The backward resample sets ``closed`` to ``'right'`` by default
+            # since the last value should be considered as the edge point for
+            # the last bin. When origin in "end" or "end_day", the value for a
+            # specific ``cftime.datetime`` index stands for the resample result
+            # from the current ``cftime.datetime`` minus ``freq`` to the current
+            # ``cftime.datetime`` with a right close.
+            self.origin in ["end", "end_day"]
+        ):
```

Comment:
```suggestion
            self.origin in ["end", "end_day"]
        ):
            # The backward resample sets ``closed`` to ``'right'`` by default
            # since the last value should be considered as the edge point for
            # the last bin. When origin in "end" or "end_day", the value for a
            # specific ``cftime.datetime`` index stands for the resample result
            # from the current ``cftime.datetime`` minus ``freq`` to the current
            # ``cftime.datetime`` with a right close.
```
The comment is about self.close/self.label so  should be closer to it right?

I'm curious if the duplicate code was intentional? Or was it supposed to be different at some point?
Anyone remembers?